projects/plugins/jetpack/changelog/update-site-logo-block-compatibility

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
From a theme developer's point of view, could this be considered a breaking change? Although if the logo is being accessed in the correct way, hopefully not

Reply:
I'm not worried about it. The site-logo feature in Jetpack is a legacy feature, mostly used in older premium themes on WP.com. From what I've seen, it's not used in themes on the dotorg repo, which usually use the Core custom-logo feature, instead.